crowbar_framework/app/controllers/api/restart_management_controller.rb

```diff
@@ -92,7 +91,7 @@ def restarts_post
       next unless node.key? :crowbar_wall
       next unless node[:crowbar_wall].key? :requires_restart
       next unless node[:crowbar_wall][:requires_restart].key? cookbook
-      next unless node[:crowbar_wall][:requires_restart][cookbook].key? service
+      next unless service.nil? || node[:crowbar_wall][:requires_restart][cookbook].key? service
```

Comment:
`next unless service.nil?` should be on its own line, probably at the start of the loop to avoid doing any checks

Reply:
Thank you

Reply:
Thank you, but I think the reason to be there is to avoid the check of the service name to reset all of them. However I am testing.